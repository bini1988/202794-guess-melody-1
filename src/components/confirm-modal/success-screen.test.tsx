import React from "react";
import renderer from "react-test-renderer";
import ConfirmModal from "./confirm-modal";


it(`ConfirmModal correctly renders default markup`, () => {
  const tree = renderer
    .create(
        <ConfirmModal
          onConfirm={() => {}}
          onCancel={() => {}}/>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
