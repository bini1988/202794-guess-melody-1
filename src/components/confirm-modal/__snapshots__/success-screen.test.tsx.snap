// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmModal correctly renders default markup 1`] = `
<section
  className="modal"
>
  <button
    className="modal__close"
    onClick={[Function]}
    type="button"
  >
    <span
      className="visually-hidden"
    >
      Закрыть
    </span>
  </button>
  <h2
    className="modal__title"
  >
    Подтверждение
  </h2>
  <p
    className="modal__text"
  >
    Вы уверены что хотите начать игру заново?
  </p>
  <div
    className="modal__buttons"
  >
    <button
      className="modal__button button"
      onClick={[Function]}
    >
      Ок
    </button>
    <button
      className="modal__button button"
      onClick={[Function]}
    >
      Отмена
    </button>
  </div>
</section>
`;
